contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -2315,10 +2315,10 @@ static void bbf_ProcessUtility(PlannedStmt *pstmt,
 							stmt->role = convertToUPN(orig_loginname);
 
 							/*
-							* Check for duplicate login
-							*/
+							 * Check for duplicate login
+							 */
 							if (get_role_oid(stmt->role, true) != InvalidOid)
-						  		ereport(ERROR, (errcode(ERRCODE_DUPLICATE_OBJECT), 
+								ereport(ERROR, (errcode(ERRCODE_DUPLICATE_OBJECT),
```

Comment:
Unnecessary change.

Reply:
Maybe it came up during resolving merge conflicts.

Reply:
This change is there to fix indentation.